### ISSUE TITLE
make metadata more robust (description)

### ISSUE THREAD

**Issue by leanne-fan** [DHRI-Curriculum/DHRIFT#71]
*make friendly colors on the highlight!!

**Comment by szweibel** [comment]
I don't know what this is referring to @leanne-fan

**Comment by leanne-fan** [comment]
@szweibel haha did we end up talking about this yesterday? I think this might have been about the metadata of each workshop having a more standard presentation?

**Comment by szweibel** [comment]
Yeah but there’s not a specific action to take, so we would need to talk
about it.

**Comment by leanne-fan** [comment]
- [ ] Currently: Complicated YAML
- [ ] Expected: Less complicated YAML
- [ ] What good looks like: turning workshop metadata into something that won't break the site
- [ ] Make it easier for workshop developers to write
